util/src/test/java/org/hyperledger/besu/util/number/PositiveNumberTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
Well spotted for noticing the copyright was required!
I think it changed quite recently (some docs may be out of date?) so it should now read "Copyright Hyperledger Besu Contributors."